scripts/zones/Oldton_Movalpolos/npcs/Scrawled_Writing.lua

```diff
@@ -3,27 +3,31 @@
 --  NPC: Scrawled_Writing
 -- Allows players to spawn NM Goblin Wolfman
 -----------------------------------
```

Comment:
Should revert all changes to this file, because it's undoing code we want.  All you want to do from the latest revision is add after the SpawnMob, a single line to make the npc disappear ...

```
        npc:setStatus(STATUS_DISAPPEAR);
```

... and up top, the require that defines STATUS_DISAPPEAR:

```
require("scripts/globals/status");
```